### PR TITLE
Stops lizard's snouts sticking through skinsuit helmets [READY]

### PR DESCRIPTION
**Pull request by Pontenerd**
## About The Pull Request

Stops lizard's snouts sticking through skinsuit helmets

## Why It's Good For The Game

It looked dumb to have your snout sticking through your skinsuit helmet
Fixes:  #1250

## Changelog
:cl:

fix: fixed lizard snouts sticking through skinsuit helmets.

/:cl:

### CODE REVIEW

File: code/modules/clothing/spacesuits/miscellaneous.dm [hunk 1]
```diff
@@ -425,8 +425,8 @@ Contains:
 	heat_protection = NONE
 	flash_protect = 0
 	bang_protect = 0
-	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEHAIR|HIDEFACIALHAIR
-	clothing_flags = STOPSPRESSUREDAMAGE | SHOWEROKAY | SNUG_FIT
```
Comment (Jalleo) [inline code/modules/clothing/spacesuits/miscellaneous.dm:429]: You have changed other aspects that might still have flags. If there is a showerokay flag or one of a similar style please re-add it.

**Review by Jalleo — CHANGES_REQUESTED** [review]
Minor problem that needs potential correcting. If its automatic with that flag then I will switch to a approval.

File: code/modules/clothing/spacesuits/miscellaneous.dm (hunk 1 above)
Comment (Pontenerd) [reply code/modules/clothing/spacesuits/miscellaneous.dm:429]: I may be stupid, will fix it now

**Comment by Pontenerd** [comment]
FUCK i changed a file that i did not mean to hold on

**Review by Kmc2000 — APPROVED** [review]

**Comment by Pontenerd** [comment]
Alrighty, this PR seems to be ready.

**Comment by Pontenerd** [comment]
Alright, this PR didn't fix the issue  AFAK as the lizard snout i used to test and the snout that causes issues are diffrent, will look into this at a later date

**Comment by Pontenerd** [comment]
I don't know how to fix this

**Comment by Pontenerd** [comment]
I don't know how to fix the issue, will attempt a fix at a later date.

**Pull request closed without merging**